tracing-effectful/tracing-effectful.cabal

```diff
@@ -33,7 +33,7 @@ library
     , base            <5
     , effectful-core  >=1.0.0.0 && <3.0.0.0
     , exceptions      <0.11
-    , tracing         >=0.0.7   && <1.0.0.0
+    , tracing         <=1.0.0.0
```

Comment:
_Nitpick:_ no upper bound?

Reply:
You mean lower ? (To be honest, it's mostly to satisfy the CI, it should probably `==` here)

Reply:
🙈 Yes :)
In that case it's just fine
BTW Cabal now also allows `1.0.*` I think

Reply:
I've put `==` (and the bound checker of the CI seems ok with it :relieved:), as this is directly linked to the library, I think we should use an exact version for now.